apps/avatax/src/pages/api/webhooks/order-confirmed.ts

```diff
@@ -61,6 +61,14 @@ export default wrapWithLoggerContext(
             return webhookResponse.error(error);
           }
 
+          if (
+            payload.order.channel.taxConfiguration?.taxCalculationStrategy ===
```

Comment:
after rebase this if can be moved to https://github.com/saleor/apps/blob/main/apps/avatax/src/modules/saleor/order.ts#L5 and taken from data instead of payload (you can have `isCalculatedWithFlatRates` method)